components/landing/TheQuickStart/StartLocally.vue

```diff
@@ -2,7 +2,13 @@
   <section class="start-locally">
     <h3>Start locally</h3>
     <p class="start-locally__introduction">
-      To install Qiskit locally, you will need
+      To
```

Comment:
Thank you for this PR @garrison 

I think this looks good. For anyone else who's coming to see this update, since the PR didn't produce a preview, this is what it looks like:

Uploading Screen Recording 2022-01-26 at 12.37.20 PM.mov…